### ISSUE TITLE
restock from containers

### ISSUE THREAD

**Issue by pizzav-xyz** [rockerle/LitematicaMapartBot#1]
can you make it restock from containers like shulker barrel chest?

**Comment by rockerle** [comment]
Yes, but I can't give an ETA when that feature will be working. Could be tomorrow, but could also be next month or next year.

**Comment by pizzav-xyz** [comment]
@rockerle can you please try

**Comment by rockerle** [comment]
To give you a little update I pushed the branch "storage". For now it contains buttons in the UI of chests, barrels and shulker boxes to add a storage block to a list of storages to memorize. 
What's missing now is the interaction with the storage blocks to pick the wanted blocks and put it in the player inventory and to incorporate that into the building routine. 
Yet again no ETA for further progress, but this project is not totally forgotten by me. Hence why I won't just close this issue. 
![Su1mMOu](https://github.com/user-attachments/assets/5807fb3e-e371-48d2-94e5-46cfa68173a9)

**Comment by pizzav-xyz** [comment]
Ooh wow thanks I didn't notice
